### PR TITLE
Fixed broken link in _docker.py

### PR DESCRIPTION
**Pull request by ArnabRaxit**
##### SUMMARY
There is a broken link in this page: https://docs.ansible.com/ansible/2.5/modules/docker_module.html
This PR fixes the broken link.
The broken link was: https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/guide_docker.rst.
Relaced that with: https://docs.ansible.com/ansible/latest/scenario_guides/guide_docker.html

##### ISSUE TYPE

- Docs Pull Request

### CODE REVIEW

**Comment by danihodovic** [comment]
LGTM

**Comment by felixfontein** [comment]
Please note that this PR is for a branch (stable-2.5) which does get seldom releases (see [here](https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html#release-status) for an overview). It could happen that this PR will not be merged because of that, but simply closed, depending on what the release manager decides.

In any case, if there is a chance to get this merged, I wouldn't mind it to be merged :-)

File: lib/ansible/modules/cloud/docker/_docker.py
Comment (mattclay) [inline lib/ansible/modules/cloud/docker/_docker.py:28]: This needs to be fixed in the latest branch in which it still exists (stable-2.6) and then backported to older branches once the fix is merged.

Also, links to our docs shouldn't be using an URL. I believe this is the correct syntax:

```suggestion
    visit our L(Getting Started with Docker Guide,.../scenario_guides/guide_docker.html).
```

@acozine, can you confirm?

**Review by mattclay — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/docker/_docker.py
Comment (acozine) [reply lib/ansible/modules/cloud/docker/_docker.py:28]: @mattclay confirmed - the correct formats are documented on https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#linking-within-module-documentation.

**Comment by acozine** [comment]
@ArnabRaxit thank you for catching that broken link and for taking the time to fix it. As @mattclay said, this problem also exists in the 2.6 version - can you fix it there and then backport the fix once it has been merged to `stable-2.6`? There are instructions on opening backport PRs at https://docs.ansible.com/ansible/latest/community/development_process.html#backport-pull-request-process or you can ask questions in the #ansible-docs channel on freenode IRC (see also https://docs.ansible.com/ansible/latest/community/communication.html#irc-channels). Thanks!

**Comment by ArnabRaxit** [comment]
Thanks @acozine , @mattclay . I have created a new PR #50950 for merging to stable-2.6 branch.

**Comment by mattclay** [comment]
@ArnabRaxit Thanks. I'll close this PR. Once https://github.com/ansible/ansible/pull/50950 is merged a backport can be created.

**Pull request closed without merging**
